css/css-masking/clip-path-svg-content/clip-path-content-syling.svg

```diff
@@ -9,8 +9,8 @@
 	<desc class="assert">Style properties on content elements of clipPath
 	must be ignored. A green square should be visible.</desc>
 </g>
-<clipPath id="clip1" clip-path="url(#clip1)">
+<clipPath id="clip1">
 	<rect x="50" y="50" width="100" height="100" stroke="black" stroke-wdith="10" stroke-dasharray="10 10" fill="none"/>
```

Comment:
Maybe fix the typo'd stroke-width (which could be relevant to this test based on the description.)

```suggestion
	<rect x="50" y="50" width="100" height="100" stroke="black" stroke-width="10" stroke-dasharray="10 10" fill="none"/>
```

Reply:
Good point! might as well fix that while we're here.